aggregator/src/helpers/networkConfig.ts

```diff
@@ -0,0 +1,9 @@
+// TODO (merge-ok) Replace this with client package import
+import { NetworkConfig, getConfig } from "../../../clients/src/NetworkConfig.ts";
+import * as env from "../env.ts";
+
+export type { NetworkConfig };
+
+export default function getNetworkConfig(): Promise<NetworkConfig> {
+    return getConfig(env.NETWORK_CONFIG_PATH, Deno.readTextFile);   
+}
```

Comment:
I don't think it's ok to reach into `../../../clients`, even temporarily. It's confusing and lowers the bar for others to add more imports like these, which would create a mess that might never be untangled.

Especially since it isn't providing much - the `NetworkConfig` type is simply cast onto the JSON file without checking, and the only fields the aggregator is actually interested in are the addresses for `VerificationGateway` and the reward token.

Given that, how about this instead?

```ts
import * as env from "../env.ts";

const uncheckedNetworkConfig = JSON.parse(
  await Deno.readTextFile(env.NETWORK_CONFIG_PATH),
);

// Exporting allows for requireAddress('testToken') in tests
export function requireAddress(name: string): string {
  const value = uncheckedNetworkConfig.addresses[name];

  if (typeof value !== "string") {
    throw new Error(`Invalid ${name} address in network config`);
  }

  return value;
}

export const verificationGatewayAddress = requireAddress("verificationGateway");
export const rewardTokenAddress = requireAddress("rewardToken");
```

Reply:
Also, although this is moot if you decide to accept the suggestion above, I feel rather strongly that if a module has a default export, then the filename of that module should match that name exactly. I'd probably resolve that by making `getNetworkConfig` a named export only, although renaming the file to `getNetworkConfig.ts` would also avoid breaking the rule.

While I'm at it 😅, there's another relevant convention I'd like to raise here. I haven't seen it much from other developers, but I believe that when a function returns a variable that is naturally named `someValue`, that function should be called `SomeValue`. This avoids the proliferation of naming prefixes like `getX`, `makeX`, `generateX`, etc., although it could also just suffer from the [one-universal-standard](https://xkcd.com/927/) problem. What are your thoughts? If pascal case functions aren't preferred, we should probably work towards removing the existing uses of it.

Reply:
Also, incorrect indentation size in this file.

Reply:
I used the clients import to get things working, and am fine switching it to the npm import. That may be worth splitting off the config work into a separate PR that is merged before this one. The idea behind having the common config is that anyone who wants to write an app/service can just point at a file/url and get all of the needed deployed contract addresses and metadata. They just need to provide the the i/o path to the file. This will be more info than is needed than by the aggregator or extension, but may be useful in other use cases.

Makes sense to validate the fields to ensure everything is provided correctly, I can do that right after it is parsed.

As far as naming goes, I generally like to have my types (structs) separate from the functions that create them. Can you provide an example of what the `NetworkConfig` func and type would look like in your naming scheme?